mrp_production_real_costs/models/stock_move.py

```diff
@@ -46,15 +46,10 @@ def create_produce_cost_line(self, amount):
                     analytic_line_obj.create(vals)
 
     @api.multi
-    def action_consume(self, product_qty, location_id=False,
-                       restrict_lot_id=False, restrict_partner_id=False,
-                       consumed_for=False):
+    def action_done(self):
         task_obj = self.env['project.task']
         analytic_line_obj = self.env['account.analytic.line']
-        result = super(StockMove, self).action_consume(
-            product_qty, location_id=location_id,
-            restrict_lot_id=restrict_lot_id,
-            restrict_partner_id=restrict_partner_id, consumed_for=consumed_for)
+        result = super(StockMove, self).action_done()
         for record in self:
             price = record.product_id.cost_price
```

Comment:
Cuidado, aquí hay que comprobar que el movimiento venga de una OF. No todos los stock.move vienen de una OF. Así estarías registrando costes para salidas, recepciones...